.github/sandbox_workflows.md

```
@@ -112,19 +72,14 @@
-PAT_WITH_REPO_PERMISSIONS
-A PAT with repo level permissions for dispatching repository events.
+### Deletion Variables Setup
 
-  Name: PAT_WITH_REPO_PERMISSIONS
-  Value: Your Personal Access Token string
+- Navigate to **Settings > Secrets and variables > Actions > Variables** in your GitHub organization.
+- Add the following variables:
+  - `TEST_AWS_ACCOUNT_ID`: The ID of the AWS account for token rotation in the test environment.
+  - `PROD_AWS_ACCOUNT_ID`: The ID of the AWS account for token rotation in the prod environment.
+  - `AWS_REGION`: The region of the AWS account.
+  - [`GITHUB_TOKEN`](https://github.com/VilnaCRM-Org/website-infrastructure/blob/main/.github/github-token-usage.md): The token for getting PR number.
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 150

(MD013, line-length)